dags/longitudinal.py

```diff
@@ -75,9 +75,9 @@
                       job_name="Distribution Viewer",
                       owner="chudson@mozilla.com",
                       email=["telemetry-alerts@mozilla.com", "chudson@mozilla.com"],
-                      execution_timeout=timedelta(hours=10),
+                      execution_timeout=timedelta(hours=12),
                       release_label="emr-5.0.0",
-                      instance_count=5,
+                      instance_count=30,
```

Comment:
I am surprised that this job takes that long. Has your job been reviewed by anyone?